CHANGELOG.md

```diff
@@ -14,7 +14,7 @@ and `a-btn__icon-on-right` classes referenced in docs to correct
 `a-btn_icon__on-left` and `a-btn_icon__on-right` classes.
 
 ### Removed
--
+- **capital-framework:** [MINOR] Remove IE7 and below support.
```

Comment:
I'm pretty sure we need to list the modified components individually to ensure they get bumped. Is that right, @contolini?